scala/private/rule_impls.bzl

```diff
@@ -1293,3 +1347,6 @@ def _jacoco_offline_instrument(ctx, input_jar):
 
 def _jacoco_offline_instrument_format_each(in_out_pair):
     return (["%s=%s" % (in_out_pair[0].path, in_out_pair[1].path)])
+
+def _is_windows(ctx):
+    return ctx.configuration.host_path_separator == ";"
```

Comment:
no better way to do this in bazel? :(

Reply:
See the comment above. `select`s cannot be used as default values for ctx attributes and this is the closest I've found in `ctx`.

Reply:
we should file a bug on this issue. This is bad news.